src/components/SwitchInput/SwitchInput.tsx

```diff
@@ -56,15 +60,16 @@ const SwitchOption = styled.div<{ isActive: boolean; disabled: boolean; percenta
   min-height: 34px;
   line-height: 34px;
 
-  ${({ isActive, disabled }) => !isActive && !disabled && `
+  ${({ isActive, disabled }) =>
+    !isActive &&
+    !disabled &&
+    `
     cursor: pointer;
-
-    &:hover {
-      text-decoration: underline;
-    }
   `}
 
-  ${({ isActive, theme }) => isActive && `
+  ${({ isActive, theme }) =>
+    isActive &&
+    `
```

Comment:
was ok
```suggestion
  ${({ isActive, theme }) =>  isActive && `
```

Reply:
Auto format did it, no Idea how it got turned on, I'll revert this.